frontend/tests/acceptance/timed-subscription-project-detail-test.js

```diff
@@ -1,50 +1,49 @@
-import { test } from 'qunit'
-import moduleForAcceptance from 'customer-center/tests/helpers/module-for-acceptance'
+import { module, test } from 'qunit'
+import { setupApplicationTest } from 'ember-qunit'
 import {
   authenticateSession,
   invalidateSession
 } from 'customer-center/tests/helpers/ember-simple-auth'
+import startApp from '../helpers/start-app'
+import destroyApp from '../helpers/destroy-app'
 
-//TODO refactor this test to https://github.com/emberjs/rfcs/blob/master/text/0232-simplify-qunit-testing-api.md
-moduleForAcceptance('Acceptance | Sysupport', {
-  application: null,
-  beforeEach() {
-    let user = server.create('user')
-    authenticateSession(this.application, { data: user.id })
-  },
-  afterEach() {
-    invalidateSession(this.application)
-  }
-})
+module('Acceptance | Sysupport Subscriptions', function(hooks) {
+  setupApplicationTest(hooks)
 
-test('subscription-project detail', function(assert) {
-  server.create('timed-subscription-project')
-  visit('/sysupport/projects')
-
-  andThen(function() {
-    click('[data-test-detail-link="0"]')
-    andThen(() => {
-      assert.equal(currentURL(), '/sysupport/projects/1')
-      assert.equal(find('[data-test-project-report]').length, 5)
-      assert.equal(find('[data-test-project-order]').length, 5)
-    })
+  hooks.beforeEach(function() {
+    this.application = null
   })
-})
 
-test('subscription-project reload', function(assert) {
-  server.create('timed-subscription-project')
-  server.create('timed-subscription-package')
-  visit('/sysupport/projects')
-
-  andThen(function() {
-    click('[data-test-reload-link="0"]')
-    andThen(() => {
-      assert.equal(currentURL(), '/sysupport/projects/1/reload')
-      click('[data-test-project-reload="0"]')
-      andThen(() => {
-        assert.equal(currentURL(), '/sysupport/projects/1')
-        assert.equal(find('[data-test-project-order]').length, 6)
-      })
-    })
+  hooks.beforeEach(async function() {
+    this.application = startApp()
```

Comment:
`setupApplicationTest(hooks)` should handle this

Reply:
I will upgrade to the new test style once i start migrating cc to ember v3 since currently my mirage version is 4.1 and the `startMirageApp` helper is in 4.2.